ui/app/components/task-group-row.js

```diff
@@ -3,34 +3,36 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import Component from '@ember/component';
+// import Component from '@ember/component';
+import Component from '@glimmer/component';
+
 import { inject as service } from '@ember/service';
 import { computed, action } from '@ember/object';
-import { alias, oneWay } from '@ember/object/computed';
+import { alias } from '@ember/object/computed';
 import { debounce } from '@ember/runloop';
-import {
-  classNames,
-  tagName,
-  attributeBindings,
-} from '@ember-decorators/component';
-import classic from 'ember-classic-decorator';
 import { lazyClick } from '../helpers/lazy-click';
 
-@classic
-@tagName('tr')
-@classNames('task-group-row', 'is-interactive')
-@attributeBindings('data-test-task-group')
 export default class TaskGroupRow extends Component {
   @service can;
+  @service router;
 
-  taskGroup = null;
+  @alias('args.taskGroup') taskGroup;
   debounce = 500;
 
-  @oneWay('taskGroup.count') count;
+  @alias('taskGroup.count') count;
```

Comment:
Made a decision to swap this from `oneWay` to `alias` because of some noted behaviour in pressing scaling buttons and not seeing the `count` update under the previous method (until post-debounce). This seems reasonable to me but I lack some of the historical context for why we wouldn't `set` from here.